### PR TITLE
sql: add support for views

### PR DESCRIPTION
**Pull request by morgo**
master has support for views now.

### CODE REVIEW

**Comment by morgo** [comment]
@XuHuaiyu @CaitinChen PTAL

File: sql/mysql-compatibility.md [hunk 1]
```diff
@@ -155,6 +154,10 @@ Due to its distributed nature, workloads that are single-threaded may perform wo
 
     When TiDB is in the execution of loading data, by default, a record with 20,000 rows of data is seen as a transaction for persistent storage. If a load data operation inserts more than 20,000 rows, it will be divided into multiple transactions to commit. If an error occurs in one transaction, this transaction in process will not be committed. However, transactions before that are committed successfully. In this case, a part of the load data operation is successfully inserted, and the rest of the data insertion fails. But MySQL treats a load data operation as a transaction, one error leads to the failure of the entire load data operation.
     
+### Views
+
+Views in TiDB are currently non-updatable. We plan to add this functionality at a later date.
```
Comment (XuHuaiyu) [inline sql/mysql-compatibility.md]: Would `non-updatable and non-insertable` be better?

Comment (morgo) [reply sql/mysql-compatibility.md]: Yes.  I have changed it, thx

**Comment by morgo** [comment]
Let's DNM the 3.0 PRs until after DITA and version split.

**Comment by CaitinChen** [comment]
@morgo The master branch corresponds to v3.0 and 3.0 PRs do not affect DITA restructuring. I think we can merge 3.0 PRs :)

**Comment by morgo** [comment]
Superseded by https://github.com/pingcap/docs/pull/1253

**Pull request closed without merging**